spec/features/idv/doc_auth/upload_step_spec.rb

```diff
@@ -9,6 +9,7 @@
 
   before do
     sign_in_and_2fa_user
+    allow_any_instance_of(Idv::Steps::UploadStep).to receive(:mobile_device?).and_return(true)
```

Comment:
There is some stubbing of `BrowserCache` in this file that I think we should remove now that the step is no longer referencing it.